iptables/iptables.go

```diff
@@ -609,6 +611,20 @@ func iptablesHasWaitCommand(v1 int, v2 int, v3 int) bool {
 	return false
 }
 
+//Checks if an iptablse version is after 1.6.0, when --wait support second
+func iptablesWaitSupportSecond(v1 int, v2 int, v3 int) bool {
+	if v1 > 1 {
+		return true
+	}
+	if v1 == 1 && v2 > 6 {
```

Comment:
Why not just do `v2 >= 6` here? Then you can skip the next conditional...

Reply:
changed